### PR TITLE
Fixes broken links in published site.

### LINKED ISSUE #402: Broken link report: 09/15/20
I've just run [broken-link-checker](https://www.npmjs.com/package/broken-link-checker) over the local dist build to generate this report, there may well be some redirects in place for a few of these links however we should double check and update the original hrefs wherever possible.

```
Getting links from: /
├─BROKEN─ /get-started/lotus/installation.md (HTTP_404)
├─BROKEN─ /get-started/lotus/send-and-receive-fil.md (HTTP_404)
├─BROKEN─ /about-filecion/faq (HTTP_404)
├─BROKEN─ /build/onboard-the-testnet (HTTP_404)
Finished! 193 links found. 139 excluded. 4 broken.

Getting links from: /mine/spacerace
├─BROKEN─ https://reward.testnet.filecoin.io/d/P_Q13JnMz/top-miners-by-block-reward?orgId=1&refresh=25s&from=now-30m&to=now&kiosk (BLC_UNKNOWN)
├─BROKEN─ https://reward.testnet.filecoin.io/ (BLC_UNKNOWN)
├─BROKEN─ https://stats.testnet.filecoin.io/ (BLC_UNKNOWN)
Finished! 230 links found. 219 excluded. 3 broken.

Getting links from: /reference/lotus-api
├─BROKEN─ https://github.com/filecoin-project/lotus/api (HTTP_404)
Finished! 192 links found. 181 excluded. 1 broken.

Getting links from: /get-started/lotus/switch-networks/
├─BROKEN─ /get-started/lotus/switch-networks/installation (HTTP_404)
Finished! 180 links found. 179 excluded. 1 broken.

Getting links from: /mine/spacerace/
├─BROKEN─ https://reward.testnet.filecoin.io/d/P_Q13JnMz/top-miners-by-block-reward?orgId=1&refresh=25s&from=now-30m&to=now&kiosk (HTTP_undefined)
├─BROKEN─ https://reward.testnet.filecoin.io/ (HTTP_undefined)
├─BROKEN─ https://stats.testnet.filecoin.io/ (HTTP_undefined)
Finished! 230 links found. 227 excluded. 3 broken.

Getting links from: /mine/lotus/miner-configuration/
├─BROKEN─ /mine/build/lotus/api-token-generation (HTTP_404)
Finished! 207 links found. 206 excluded. 1 broken.

Getting links from: /mine/lotus/replace-messages-in-mpool/
├─BROKEN─ /mine/reference/lotus-api (HTTP_404)
Finished! 179 links found. 176 excluded. 1 broken.

Getting links from: /mine/lotus/seal-workers/
├─BROKEN─ /mine/build/lotus/api-token-generation (HTTP_404)
Finished! 202 links found. 201 excluded. 1 broken.

Getting links from: /mine/lotus/miner-troubleshooting/
├─BROKEN─ /mine/lotus/miner-troubleshooting/miner-setup#publishing-the-miner-addresses (HTTP_404)
Finished! 203 links found. 198 excluded. 1 broken.

Getting links from: /build/filecoin-pinning-services/
├─BROKEN─ /build/build/powergate (HTTP_404)
Finished! 195 links found. 194 excluded. 1 broken.

Getting links from: /build/powergate/
├─BROKEN─ /build/powergate/examples/simple-pinning-service/overview (HTTP_404)
Finished! 194 links found. 193 excluded. 1 broken.

Getting links from: /build/lotus/payment-channels/
├─BROKEN─ /build/reference/lotus-api (HTTP_404)
Finished! 186 links found. 183 excluded. 1 broken.

Getting links from: /build/examples/simple-pinning-service/powergate-lotus-go-ipfs-interactions/
├─BROKEN─ /build/about-filecoin/what-is-filecoin (HTTP_404)
Finished! 199 links found. 191 excluded. 1 broken.

Getting links from: /reference/lotus-api/
├─BROKEN─ /reference/get-started/lotus/configuration-and-advanced-usage (HTTP_404)
├─BROKEN─ /reference/mine/lotus/miner-configuration (HTTP_404)
├─BROKEN─ https://github.com/filecoin-project/lotus/api (HTTP_404)
├─BROKEN─ /reference/build/lotus/api-client-libraries (HTTP_404)
├─BROKEN─ /reference/build/lotus/go-json-rpc (HTTP_404)
Finished! 192 links found. 187 excluded. 5 broken.

Getting links from: /build/examples/network-inspector/step-2-run-the-react-app/
├─BROKEN─ https://localhost:3000/ (ERRNO_ECONNREFUSED)
Finished! 190 links found. 187 excluded. 1 broken.

Getting links from: /build/examples/network-inspector/step-3-set-up-the-lotus-and-go-ipfs-api-clients/
├─BROKEN─ http://npmjs.com/package/filecoin-shipyard/lotus-client-provider-browser (HTTP_404)
Finished! 196 links found. 183 excluded. 1 broken.

Getting links from: /build/examples/meme-marketplace/step-3-run-hub-auth-server/
├─BROKEN─ https://docs.textile.io/hub/app-apis/ (HTTP_404)
Finished! 196 links found. 188 excluded. 1 broken.

Getting links from: /build/examples/meme-marketplace/step-4-connecting-app-with-auth-server/
├─BROKEN─ https://www.npmjs.com/package/koa-WebSocket (HTTP_404)
Finished! 218 links found. 206 excluded. 1 broken.

Getting links from: /build/examples/meme-marketplace/step-6-explore-app/
├─BROKEN─ /build/examples/meme-marketplace/step-5-connecting-app-with-blockchain.md/#step-5d-sending-transaction-to-register-memes-on-blockchain (HTTP_404)
├─BROKEN─ /build/examples/meme-marketplace/step-5-connecting-app-with-blockchain.md/#step-5e-sending-calls-to-fetch-meme-details-form-the-blockchain (HTTP_404)
Finished! 209 links found. 198 excluded. 2 broken.

Finished! 32719 links found. 32009 excluded. 31 broken.
Elapsed time: 3 minutes, 34 seconds
```

### PR DESCRIPTION
**Pull request by johnnymatthews**
There's a bunch of broken links in #402. This PR fixes them.

Closes #402.

### CODE REVIEW

File: docs/build/examples/meme-marketplace/step-3-run-hub-auth-server.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Before starting the setup for the authentication server, you need to create API
 To do that, you’ll need:
 
 - [An account](https://docs.textile.io/hub/accounts/). This is your developer account on the Hub.
-- [A user group key](https://docs.textile.io/hub/app-apis/). This is how your users will be able to access your Hub APIs. Consider creating the key in an organization, not as your personal account, so that you can invite collaborators later. You only need to create one _user group key_ for all the users of your app.
+- [A user group key](https://docs.textile.io/hub/apis/). This is how your users will be able to access your Hub APIs. Consider creating the key in an organization, not as your personal account, so that you can invite collaborators later. You only need to create one _user group key_ for all the users of your app.
```
Comment (jimmylee) [inline docs/build/examples/meme-marketplace/step-3-run-hub-auth-server.md:20]: `organization, not as your personal account` -> `organization account, not in your personal account`?

File: docs/build/examples/network-inspector/step-2-run-the-react-app.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ npm ci
 npm start
 ```
 
-This starts the Filecoin network inspector app and opens the application in your browser at [https://localhost:3000](https://localhost:3000).
+This starts the Filecoin network inspector app and opens the application in your browser at `localhost:3000`.
```
Comment (jimmylee) [inline docs/build/examples/network-inspector/step-2-run-the-react-app.md:30]:
```
This starts the Filecoin network inspector app and opens the application in your browser at `localhost:3000`.
```

maybe

```
This starts and opens the Filecoin network inspector app in your browser at `localhost:3000`
```

**Review by jimmylee — APPROVED** [review]
LGTM, from what I can see.

**Comment by cwaring** [comment]
👌

**Review by mishmosh — APPROVED** [review]
LGTM. Also created #405.

**Comment by cwaring** [comment]
Also the coincidence of this PR number being #404 is something to be celebrated 😂🥳

**Comment by mishmosh** [comment]
It's poetry.

File: docs/build/examples/meme-marketplace/step-3-run-hub-auth-server.md (hunk 1 above)
Comment (johnnymatthews) [reply docs/build/examples/meme-marketplace/step-3-run-hub-auth-server.md:20]: I'm not touching copy in this PR. Literally just fixing links.

File: docs/build/examples/network-inspector/step-2-run-the-react-app.md (hunk 2 above)
Comment (johnnymatthews) [reply docs/build/examples/network-inspector/step-2-run-the-react-app.md:30]: Again, I'm not touching copy in this PR. Literally just fixing links.

**Pull request merged**